HPXMLtoOpenStudio/tests/test_battery.rb

```diff
@@ -125,7 +125,43 @@ def test_pv_battery
       # Check object
       assert(!battery.thermalZone.is_initialized)
       assert_equal(0, battery.radiativeFraction)
-      assert_equal(0.95, battery.dctoDCChargingEfficiency)
+      assert_equal(0.925, battery.dctoDCChargingEfficiency)
+      assert_equal(HPXML::BatteryLifetimeModelNone, battery.lifetimeModel)
+      assert_in_epsilon(15, battery.numberofCellsinSeries, 0.01)
+      assert_in_epsilon(125, battery.numberofStringsinParallel, 0.01)
+      assert_in_epsilon(0.0, battery.initialFractionalStateofCharge, 0.01)
+      assert_in_epsilon(198.0, battery.batteryMass, 0.01)
+      assert_in_epsilon(2.25, battery.batterySurfaceArea, 0.01)
+      assert_in_epsilon(20000, calc_nom_capacity(battery), 0.01)
+
+      elcds = model.getElectricLoadCenterDistributions
+      assert_equal(1, elcds.size)
+      elcd = elcds[0]
+      assert_equal('DirectCurrentWithInverterACStorage', elcd.electricalBussType)
+      assert_equal(0.075, elcd.minimumStorageStateofChargeFraction)
+      assert_equal(0.975, elcd.maximumStorageStateofChargeFraction)
+      assert_equal(6000.0, elcd.designStorageControlChargePower.get)
+      assert_equal(6000.0, elcd.designStorageControlDischargePower.get)
+      assert(!elcd.demandLimitSchemePurchasedElectricDemandLimit.is_initialized)
+      assert_equal('TrackFacilityElectricDemandStoreExcessOnSite', elcd.storageOperationScheme)
+      assert(!elcd.storageChargePowerFractionSchedule.is_initialized)
+      assert(!elcd.storageDischargePowerFractionSchedule.is_initialized)
+      assert(!elcd.storageConverter.is_initialized)
+    end
+  end
+
+  def test_pv_battery_shared
```

Comment:
New test.

Reply:
Also good to change the default efficiency to match our OS-HPXML, good catch that it's not actually 0.95.